src/main/java/com/conveyal/datatools/manager/jobs/ValidateFeedJob.java

```diff
@@ -44,13 +44,7 @@ public void jobFinished () {
                 // such as BuildTransportNetwork, to finish. If those subsequent jobs fail,
                 // the version won't get loaded into MongoDB (even though it exists in postgres).
                 feedVersion.storeUser(owner);
-                if (isNewVersion) {
-                    int count = feedVersion.parentFeedSource().feedVersionCount();
-                    feedVersion.version = count + 1;
-                    Persistence.feedVersions.create(feedVersion);
-                } else {
-                    Persistence.feedVersions.replace(feedVersion.id, feedVersion);
-                }
+                feedVersion.persistFeedVersionAfterValidation(isNewVersion);
```

Comment:
As written, it looks like the new version is shown in the UI (on the `mobility-data-validator` branch) after the first validator completes. Is that the desired effect?

Reply:
I think that's ok as long as we refresh the UI when the mobility data results come in, which is something we can do on the client.